tests/API/XCCDF/unittests/test_fix_arf.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+profile="xccdf_moc.elpmaxe.www_profile_standard"
+result_id="xccdf_org.open-scap_testresult_xccdf_moc.elpmaxe.www_profile_standard"
+bash_line1="echo this_is_ok"
+bash_line2="echo fix_me_please"
+ansible_template="urn:xccdf:fix:script:ansible"
+ansible_line1="this_is_ok"
+ansible_line2="fix_me_please"
```

Comment:
@dahaic Yes, I can, that would be definitely better. What do you think should there be a valid ansible playbook or just an arbitrary string? Personally I would prefer a valid ansible playbook but I'm afraid that then it would not be an atomic value.